### PR TITLE
Pin the History button to every breakpoint

### PR DESCRIPTION
**Pull request by freekh**
Test only — no behaviour change.

The History button is already offered on mobile and tablet: it sits **outside** the `!isMobile` guards in the top bar. But nothing said so, and the four buttons immediately around it are each behind one — Review, Preview, Publish and Quick actions.

A refactor that widened one of those guards over this button would take history away from phones and tablets **without failing anything**. The desktop screenshot would look untouched, and the only symptom is a feature that silently does not exist on an iPad.

## What's pinned

The breakpoints, not the layout:

- offered at `mobile`, `tablet` and `desktop`
- hidden at all three when `historyEnabled` is false — fs mode, where there is no published history and the button could only open an apology

Verified by putting it behind `!isMobile`: the mobile case goes red.

## Context

The rest of the responsive behaviour was already in place and this doesn't change it — recorded here so the test explains itself:

- the panel is `mobileVariant="bottom-sheet"`
- `HistorySplit` turns the two panes into `Now` / the commit as **tabs** below desktop, keeping both mounted so scroll position and half-typed edits survive toggling
- breakpoints are mobile `< 768px`, tablet `768–1199px`, desktop `≥ 1200px` — so an iPad in portrait is tablet, and a 12.9" iPad Pro in landscape is desktop

`jsdom` has no `matchMedia` and the non-desktop top bar reaches it through `usePrefersReducedMotion`, so the test stubs it — otherwise the mobile and tablet cases fail for a reason unrelated to the button.

Typecheck, prettier and eslint clean.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://claude.ai/code/session_01X1XGRuPC6BTaMcxizVttCb

---
_Generated by [Claude Code](https://claude.ai/code/session_01X1XGRuPC6BTaMcxizVttCb)_

### CODE REVIEW

File: packages/ui/spa/components/shell/historyAffordance.test.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  );
+}
+
+const BREAKPOINTS: ShellBreakpoint[] = ["mobile", "tablet", "desktop"];
+
+describe("the History button", () => {
+  test.each(BREAKPOINTS)("is offered on %s", (breakpoint) => {
+    render(topBar(breakpoint, true));
+    expect(screen.getByRole("button", { name: "History" })).not.toBeNull();
+  });
```
[the remark this answers is not in the document]
Comment (freekh) [reply packages/ui/spa/components/shell/historyAffordance.test.tsx:66]: Technically right — `getByRole` throws, so the assertion never fires. Keeping it anyway, for consistency: this exact idiom is used **15 times** across `packages/ui/spa`'s tests (`panelRow.test.tsx`, `aiAffordances.test.tsx`, and others), and changing only the new file would make it the odd one out among sixteen.

On the "obscures the failure signal" half: it doesn't, in practice. When the button is missing, the throw from `getByRole` is what reports — with testing-library's message listing the roles it did find, which is a better failure than a bare `expected null not to be null` would be. The `.not.toBeNull()` is redundant rather than misleading.

There's no `jest/expect-expect` rule configured, so dropping the `expect` would lint fine — this is purely a house-style call, and the house style is the one already written down 15 times.

---
_Generated by [Claude Code](https://claude.ai/code)_
